datafusion/proto/src/logical_plan/from_proto.rs

```diff
@@ -46,7 +46,7 @@ use datafusion_expr::{
     array_to_string, ascii, asin, asinh, atan, atan2, atanh, bit_length, btrim,
     cardinality, cbrt, ceil, character_length, chr, coalesce, concat_expr,
     concat_ws_expr, cos, cosh, cot, current_date, current_time, date_bin, date_part,
-    date_trunc, degrees, digest, exp,
+    date_trunc, decode, degrees, digest, encode, exp,
```

Comment:
BTW during review I found a few more functions that need the same treatment, so I filed https://github.com/apache/arrow-datafusion/issues/8098